### PR TITLE
fix(android, exoplayer): ensure playback progress is reported

### PR DESCRIPTION
**Pull request by freeboub**
fix(android, exoplayer): ensure playback progress is reported even if player is paused

Adresses #2141 on Android.
and shall fix: https://github.com/react-native-video/react-native-video/pull/2372

#### Provide an example of how to test the change
A live channel sample is available in basic sample.
On live stream, position indicator keeps updating after playback is paused.

#### Focus area
You can ensure that position is not reported too often

#### Describe the changes
- Don't test player state before sending time updates
- safety check to avoid sending the same value multiple times (avoid UI updates)

### CODE REVIEW

**Comment by hueniverse** [comment]
Thanks! As soon as we get it reviewed it will be merged!

File: android-exoplayer/src/main/java/com/brentvatne/exoplayer/ReactExoplayerView.java [hunk 1]
```diff
@@ -165,21 +165,30 @@ class ReactExoplayerView extends FrameLayout implements
...
+    private long lastPos = -1;
+    private long lastBufferDuration = -1;
+    private long lastDuration = -1;
+
     private final Handler progressHandler = new Handler() {
         @Override
         public void handleMessage(Message msg) {
             switch (msg.what) {
                 case SHOW_PROGRESS:
-                    if (player != null
```
Comment (armands-malejevs) [inline android-exoplayer/src/main/java/com/brentvatne/exoplayer/ReactExoplayerView.java]: Why did you remove this null check? Wouldn't it be safer to leave it there? (I'm OK with removing the state checks, but null check might be beneficial here).

**Review by armands-malejevs — CHANGES_REQUESTED** [review]

File: android-exoplayer/src/main/java/com/brentvatne/exoplayer/ReactExoplayerView.java (hunk 1 above)
Comment (freeboub) [reply android-exoplayer/src/main/java/com/brentvatne/exoplayer/ReactExoplayerView.java]: I fully agree ! change done

**Review by armands-malejevs — APPROVED** [review]

**Comment by freeboub** [comment]
Just fix conflict, ready to merge
